README.md

```diff
@@ -7,20 +7,22 @@
 BRP Bevragen is een Haal Centraal API voor het zoeken en raadplegen van ingeschreven natuurlijke personen voor alle binnengemeentelijke afnemers in NL. De informatie die de API levert is herleidbaar naar het LO GBA 3.10.
 
 # Planning
+Release v1.0.0 is uitgebracht en is [hier](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/tree/v1.0.0) te vinden op 
+
 https://eu-rm.roadmunk.com/publish/29a13c572a0dfc79f1d0386fc572e15383afe11f
 
 ## Getting started
 Om te beginnen met ontwikkelen van de API is de [getting started](./docs/getting-started.md) een goed beginpunt. De API is technisch gespecificeerd in Open API specificaties (zie hieronder bij documentatie).
 
 ## Documentatie
-* [Technische specificaties](./specificatie) (Open API Specificaties en JSON schema) en in [Swagger-formaat](https://petstore.swagger.io/?url=https://raw.githubusercontent.com/VNG-Realisatie/Bevragingen-ingeschreven-personen/master/specificatie/openapi.yaml)
-* [Productvisie](https://github.com/VNG-Realisatie/RSGB-bevragingen/blob/master/docs/productvision.md)
-* [Definition of Ready](https://github.com/VNG-Realisatie/RSGB-bevragingen/blob/master/docs/definition_of_ready.md)
-* [Definition of Done](https://github.com/VNG-Realisatie/RSGB-bevragingen/blob/master/docs/definition_of_done.md)
-* Hoe u kunt [bijdragen](https://github.com/VNG-Realisatie/Tutorial/blob/master/CONTRIBUTING.md)
-* [Omgangsvormen](https://github.com/VNG-Realisatie/Tutorial/blob/master/CODE_OF_CONDUCT.md)
+* [Technische specificaties](./specificatie) (Open API Specificaties en JSON schema) en in [Swagger-formaat](https://petstore.swagger.io/?url=https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/master/specificatie/genereervariant/openapi.yaml)
+* [Productvisie](./docs/productvision.md)
+* [Definition of Ready](./docs/definition_of_ready.md)
+* [Definition of Done](./docs/definition_of_done.md)
+* Hoe u kunt [bijdragen](https://github.com/VNG-Realisatie/API-kennisbank/blob/master/CONTRIBUTING.md)
+* [Omgangsvormen](https://github.com/VNG-Realisatie/API-kennisbank/blob/master/CODE_OF_CONDUCT.md)
 * [Functionele specificaties](./features)
-* Ontwerpkeuzes staan in het document [Design decisions](https://github.com/VNG-Realisatie/Bevragingen-ingeschreven-personen/blob/master/docs/design_decisions.md)
+* Ontwerpkeuzes staan in het document [Design decisions](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/blob/master/docs/design_decisions.md)
```

Comment:
waarom is dit geen relatieve link, terwijl de andere links (naar productvisie, dod en dor) wel relatief zijn?